common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1083,6 +1083,12 @@
     ],
     "sqlState" : "42702"
   },
+  "EXECUTE_IMMEDIATE_FAILED_TO_PARSE_STACK_OVERFLOW" : {
```

Comment:
Why can't we have a GENERAL fix? PARSE_STACK overflow is not EXECUTE IMMEDIATE specific.

Reply:
updated